### PR TITLE
JW-1433 : Adding controller

### PR DESCRIPTION
**Pull request by kaweue**
Adding roles contrail-ansible-deployer, kolla-provision-docker and controller

### CODE REVIEW

**Comment by kaweue** [comment]
@katrybacka FYI

File: ansible/roles/controller/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- name: Install build dependencies
+  package:
+    name: "{{ item }}"
+    state: latest
+  become: true
+  with_items:
+    - python2-requests
+    - git
+
+- include_role:
```
Comment (katrybacka) [inline ansible/roles/controller/tasks/main.yml]: Can we use list of roles here instead of using ```include role``` twice?

**Review by katrybacka — DISMISSED** [review]

File: ansible/roles/controller/tasks/main.yml (hunk 1 above)
Comment (jablonskim) [reply ansible/roles/controller/tasks/main.yml]: +1

**Review by jablonskim — CHANGES_REQUESTED** [review]

**Review by katrybacka — APPROVED** [review]
MJ + KR

**Comment by kaweue** [comment]
recheck no bug

**Review by jablonskim — APPROVED** [review]

**Comment by kaweue** [comment]
recheck no bug

**Pull request merged**